### ISSUE TITLE
minify makes js bigger

### ISSUE THREAD

**Issue by sirenkovladd** [putoutjs/minify#15]
- **Version** (`npm -v`): 10.5.2
- **Node Version** `node -v`: v20.13.1
- **OS** (`uname -a` on Linux): Darwin arm64

Looks like @putout/minify makes js bigger

```js
import { minify } from "@putout/minify";

const source = `()=>{try{return 1}catch(e){return console.log(e),2}}`;
const d = minify(source);
console.log(d);

console.log(source.length, '->', d.length);
```

```sh
❯ node test.js
()=>{try {return 1} catch (e) {console.log(e);return 2}};
52 -> 57
```

**Comment by sirenkovladd** [comment]
also
```js
const source = `for (let e of l)(e.o=w(e.o)),(e.l=w(e.l))`;
```

result
```js
for(let e of l){(e.o=w(e.o));(e.l=w(e.l))}
```

expect
```js
for(let e of l)e.o=w(e.o),e.l=w(e.l)
```

**Comment by sirenkovladd** [comment]
also
```js
const source = `() => { d(); return 1 }`;
```

result
```js
()=>{d();return 1};
```

expect
```js
()=>(d(),1)
```

**Comment by coderaiser** [comment]
Thanks! Current result:

```js
()=>{try{return 1}catch(e){console.log(e);return 2}};
```

**Comment by coderaiser** [comment]
Just landed: https://github.com/coderaiser/putout/blob/master/packages/plugin-minify/README.md#convert-return-to-sequence-expression

**Comment by coderaiser** [comment]
Just updated, please try latest version of `@putout/minify`. Is it works for you?

**Comment by sirenkovladd** [comment]
2/3 works
```js
const source = `for (let e of l)(e.o=w(e.o)),(e.l=w(e.l))`;
```
does not work for this line

**Comment by coderaiser** [comment]
Landed in [`v3.19`](https://github.com/putoutjs/minify/releases/tag/v3.19.0) 🎉.

Is it works for you?

**Comment by sirenkovladd** [comment]
Almost, now I have it
```js
for(let e of l)(e.o=w(e.o)),(e.l=w(e.l));
```

but expect
```js
for(let e of l)e.o=w(e.o),e.l=w(e.l)
```

**Comment by coderaiser** [comment]
Landed in [v3.19.1](https://github.com/putoutjs/minify/releases/tag/v3.19.1) 🎉. Is it works for you?

**Comment by sirenkovladd** [comment]
![image](https://github.com/putoutjs/minify/assets/28163663/9474525e-c0a2-40f4-b1f3-b34f71ddfccb)
It looks like not

**Comment by sirenkovladd** [comment]
Also found a related error

```js
const source = `export const s = () => a ? (p(),i) : 2`
```

result
```js
export var s=()=>a?p(),i:2;
```

```sh
export var s=()=>a?p(),i:2;
                      ^

SyntaxError: Unexpected token ','
    at ModuleLoader.moduleStrategy (node:internal/modules/esm/translators:169:18)
    at callTranslator (node:internal/modules/esm/loader:272:14)
    at ModuleLoader.moduleProvider (node:internal/modules/esm/loader:278:30)
    at async link (node:internal/modules/esm/module_job:78:21)

Node.js v20.13.1
```

**Comment by coderaiser** [comment]
Could you please create another issue for another cases

**Comment by coderaiser** [comment]
Landed in [v3.19.3](https://github.com/putoutjs/minify/releases/tag/v3.19.3).

SequenceExpressions is black hole I would be happy to add more features to support every case, but it will be definitely a lot of them, maybe 20 more, who knows, let's use different issues for each syntax related case.
